scripts/run-with-strimzi.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -eo pipefail
+
+if [[ -z $QUAY_ORG ]]; then
+  echo "Please set QUAY_ORG, exiting"
+  exit 1
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. "${SCRIPT_DIR}/common.sh"
+cd "${SCRIPT_DIR}/.."
+
+if [[ "${QUAY_ORG}" != 'kroxylicious' ]]; then
+  echo "building and pushing image to quay.io"
+  PUSH_IMAGE=y "${SCRIPT_DIR}/deploy-image.sh"
+else
+  echo "QUAY_ORG is kroxylicious, not building/deploying image"
+fi
```

Comment:
Whats the goal here? Are you trying to ensure we don't push dev images to a project wide repo?

Reply:
Yes, I imagine we will want to be able to run this script in future targeting a particular tag in the `kroxylicious` quay and we don't want to accidentally push to the public quay repo.

Reply:
Maybe throw a comment in there to that effect?